data/json/furniture.json

```diff
@@ -2988,6 +2988,84 @@
             ]
         }
     },
+    {
+        "type" : "furniture",
+        "id" : "f_butcher_rack",
+        "name": "butchering rack",
+        "description": "Butchering rack designed to hang the carcass in the air.",
+        "symbol": "^",
+        "bgcolor": "brown",
+        "move_cost_mod": 2,
+        "required_str": -1,
+        "flags": ["PLACE_ITEM", "TRANSPARENT", "FLAMABLE_ASH", "MOUNTABLE", "ALLOW_FIELD_EFFECT", "EASY_DECONSTRUCT", "BUTCHER_EQ", "ORGANIC"],
+        "deconstruct": {
+            "items": [
+            { "item": "stick_long", "count": 6 }
+            ]
+        },
+        "bash": {
+            "str_min": 8, "str_max": 30,
+            "sound": "crunch!",
+            "sound_fail": "whump!",
+            "items": [
+                { "item": "stick_long", "count": [1, 6] }
+            ]
+        }
+    },
+    {
+        "type" : "furniture",
+        "id" : "f_metal_butcher_rack",
+        "name": "metal butchering rack",
+        "description": "Metal butchering rack designed to hang the carcass in the air.  It can be deconstructed and folded for easy transportation.",
+        "symbol": "^",
+        "bgcolor": "light_gray",
+        "move_cost_mod": 2,
+        "required_str": -1,
+        "deployed_item": "metal_butcher_rack",
+        "examine_action": "deployed_furniture",
+        "flags": ["PLACE_ITEM", "TRANSPARENT", "MOUNTABLE", "ALLOW_FIELD_EFFECT", "BUTCHER_EQ"],
+        "deconstruct": {
+            "items": [
+            { "item": "metal_butcher_rack", "count": 1 }
+            ]
+        },
+        "bash": {
+            "str_min": 18, "str_max": 50,
+            "sound": "metal screeching!",
+            "sound_fail": "clang!",
+            "items": [
+                { "item": "scrap", "count": [1, 3] },
+                { "item": "pipe", "count": [1, 3] }
+            ]
+        }
+    },
+    {
+        "type" : "furniture",
+        "id" : "f_tourist_table",
+        "name": "tourist table",
+        "description": "Small metal folding table, ideal for off-road trips to the wild.",
```

Comment:
Not sure if `into the wild` sounds better than `to the wild`, but a minor suggestion nonetheless.